### PR TITLE
Request PID 5284 for nRFMicro

### PR DESCRIPTION
**Pull request by joric**
The project was pretty popular for a while (1.4k stars currently). Going to have an official bootloader.

### CODE REVIEW

**Comment by pdcook** [comment]
Bootloader is a variant of Adafruit's which requires VID/PID for official integration in their library. PR here: https://github.com/adafruit/Adafruit_nRF52_Bootloader/pull/342. MIT License.

Arduino core for this board is coming soon

File: 1209/5284/index.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+layout: pid
+title: nRFMicro
```
Comment (tannewt) [inline 1209/5284/index.md:3]:
```suggestion
title: nRFMicro Bootloader

```

**Review by tannewt — CHANGES_REQUESTED** [review]
Please link to the bootloader PR from your repo because that's the software that will use this PID. You can follow up with requests for other firmware (Arduino and CircuitPython) too.

File: 1209/5284/index.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+layout: pid
+title: nRFMicro Bootloader
+owner: joric
+license: MIT
+site: https://github.com/adafruit/Adafruit_nRF52_Bootloader/pull/342
+source: https://github.com/joric/nrfmicro
+---
```
Comment (tannewt) [inline 1209/5284/index.md:8]: Please add a description here. (I'm not sure why you deleted it.)

**Review by tannewt — CHANGES_REQUESTED** [review]
Please add a description back.

File: 1209/5284/index.md (hunk 2 above)
Comment (joric) [reply 1209/5284/index.md:8]: 36.6% of records here don't have a description (734 records total, 471 records with description and 272 records without description).

**Review by tannewt — APPROVED** [review]
Descriptions are useful and I'd like to see them going forwards. I thought you'd add a note that it was for the bootloader in the description. You can do that in a follow up if you want.

**Pull request merged**